extensions/amp-analytics/0.1/events.js

```diff
@@ -230,14 +246,17 @@ export class AnalyticsEvent {
    * @param {!Element} target The most relevant target element.
    * @param {string} type The type of event.
    * @param {?JsonObject=} opt_vars A map of vars and their values.
+   * @param {boolean} enableDataVars
```

Comment:
can you add a short description of what `enableDataVars` means?

Reply:
done